src/components/FieldNumber/FieldNumber.module.css

```diff
@@ -0,0 +1,46 @@
+@import '../../styles/customMediaQueries.css';
+
+.root {
+  position: relative;
+}
+
+.numberInputWrapper {
+  align-self: baseline;
+  height: 40px;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  cursor: pointer;
+  min-width: 104px;
+}
+
+.numberButton {
+  border-width: thin;
```

Comment:
We have used pixels.

Reply:
When changing this to SVGs, border-width became stroke-width but I set that with pixels now 👍